### DISCUSSION TITLE
Better printing emulation

### DISCUSSION THREAD

**Discussion by awildfurret** [86Box/86Box#1729] [Ideas]
While the virtual printing is cool, printing banners in Bannermania or possibly other banner softwares causes the banner to split apart. But still, the virtual printing is an achievement to me

I suggest that this emulator should add more printing types or printer emulation for publishing softwares. So we can make more cool virtual banners and possibly reliving childhoods

I'm pretty sure this request might be ignored or not important but, this can improve the emulator. I would love to see the response from the devs

**Comment by dhrdlicka** [comment]
This is a rather... generic sounding request. Do you have any specific improvements in mind?

**Comment by awildfurret** [comment]
Well... Yes? I'm just hoping for more paper sizes support other than A3 and A4. And possibly banners, making banners is fun! When comes to printer models, I'm not sure for printer models, but its up to the developers. I hope 86box's virtual printing accuracy improves as time goes on

**Comment by awildfurret** [comment]
Update: Add set printing result resolution, so the size of the result is not too large

**Comment by awildfurret** [comment]
Update 2:
Add coloured papers for older publishing software?
